test/src/TapedTest.cpp

```diff
@@ -0,0 +1,58 @@
+#include <helpers/commonHelpers.hpp>
+#include <rgl/api/extensions/tape.h>
+
+#include <filesystem>
+#include <spdlog/fmt/ostr.h>
+
+#if RGL_BUILD_PCL_EXTENSION
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+#include <pcl/io/pcd_io.h>
+#endif
+
+class TapedTest : public RGLTest
+{
+protected:
+	const std::string benchmarkDataDirEnvVariableName = "RGL_TAPED_TEST_DATA_DIR";
+	const float epsilon = 1e-2f;
+};
+
+TEST_F(TapedTest, compare_pcd_files)
```

Comment:
```suggestion
TEST_F(TapedTest, awsim-mesh2pcd)
```
I would specify what the use-case the test validates.